### DISCUSSION TITLE
v1.0.0-alpha1

### DISCUSSION THREAD

**Discussion by wiredopposite** [wiredopposite/OGX-Mini#49] [Announcements]
**Features new to v1.0.0-alpha release**
- Bluetooth functionality for the Pico W
- Web application for configuring deadzones and buttons mappings, supports up to 8 saved profiles.
- Pi Pico 2 (RP2350) support.
- Delayed USB mount until a controller is plugged in, useful for internal installation (non-Pico W only).
- Generic HID controller support (fallback for unknown HID controllers).
- Dualshock 3 emulation (minus gyros), PS3 rumble now works.
- Steel Battalion controller emulation with a wireless Xbox 360 chatpad, same mappings as OGX360.
- Analog button support on OG Xbox, PS3, and DInput.
- RGB LED support for RP2040-Zero and Adafruit Feather boards.

**Fixes**
- Fixed Xbox 360 USB descriptor issue causing some inputs to not work.
- More consistent setting of Xbox 360 LEDs on connect.
- Fixed wrong RP240-Zero LED pin in source, though the RGB LED can be used now.

**New to source**
- Updated license, wired firmware is still all MIT, Bluetooth has a mixed license for Bluepad32 and BTStack
- ESP32 firmware to pair with the RP2040, hardware diagrams and binaries will be coming shortly
- 4 channel i2c, hook up 4 RP2040-Zero boards via i2c and use one Xbox 360 wireless adapter to control all 4.
- Xbox DVD dongle emulation, dump your firmware (see Tools directory) and compile to use this feature.
- New, clearer hardware diagrams in the hardware folder.

<hr /><em>This discussion was created from the release <a href='https://github.com/wiredopposite/OGX-Mini/releases/tag/'>v1.0.0-alpha1</a>.</em>

**Comment by andres01995** [comment]
Hello friend, how are you, I apologize for my English, I am using Google Translate, I am a fan of your work, I am writing to you because I bought an OGX mini adapter to play with the new Hyperkin Dusches controller, but it does not recognize it, so I updated the adapter to the version v1.0.0-alpha1, it recognizes it perfectly but after a few seconds it disconnects, and the adapter LED, any solution? thank you so much.
